src/main/java/org/sagebionetworks/bridge/services/PermissionService.java

```diff
@@ -0,0 +1,169 @@
+package org.sagebionetworks.bridge.services;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.sagebionetworks.bridge.models.permissions.EntityType.*;
```

Comment:
Expand imports... we don't use .* as a rule

Reply:
Noted and corrected!